L1Trigger/CSCTriggerPrimitives/plugins/CSCTriggerPrimitivesAnalyzer.h

```diff
@@ -0,0 +1,72 @@
+#ifndef L1Trigger_CSCTriggerPrimitives_CSCTriggerPrimitivesAnalyzer_h
```

Comment:
Do not split plugins into header and source files.
(see https://cms-sw.github.io/cms_coding_rules.html)

Reply:
Turned into a single .cc file.